javascripts/discourse/components/featured-homepage-topics.js

```diff
@@ -92,6 +93,16 @@ export default class FeaturedHomepageTopics extends Component {
     }
   }
 
+  emojiTitle(t) {
+    return emojiUnescape(t);
+  }
+
+  topicHref(t) {
+    return `/t/${t.slug}/${t.id}/${
```

Comment:
Probably minor, but this may need a small change for subfolder compatibility. Maybe using `import getURL from "discourse-common/lib/get-url";`.